packages/@secretlint/secretlint-rule-secp256k1-priv/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Changelog
```

Comment:
We will create CHANGELOG file when publish this package automatically by CI.
I don't need you to create the CHANGELOG file.

https://github.com/secretlint/secretlint/blob/a5f56d1df9c9d88b7a8ff82867ab7e51aab06a20/.github/workflows/publish.yml#L53